server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/LogicalPlanBuilder.java

```diff
@@ -1111,4 +1115,61 @@ public LogicalPlanBuilder planPathsUsingTemplateSource(
             context.getQueryId().genPlanNodeId(), pathPatternList, templateId);
     return this;
   }
+
+  public LogicalPlanBuilder planOneChildOrderBy(
```

Comment:
MergeSort should never have only one chil, if so, there is no need to add one MergeSortNode. And BTW, we must make sure all the children have already been sorted by order item before they are sent into MergeSortOperator.